pkg/services/playlist/playlistimpl/object_store.go

```diff
@@ -27,30 +27,28 @@ type objectStoreImpl struct {
 var _ playlist.Service = &objectStoreImpl{}
 
 func (s *objectStoreImpl) sync() {
-	rows, err := s.sess.Query(context.Background(), "SELECT org_id,uid FROM playlist ORDER BY org_id asc")
+	type Info struct {
+		OrgID int64  `db:"org_id"`
+		UID   string `db:"uid"`
+	}
+	results := []Info{}
+	err := s.sess.Select(context.Background(), &results, "SELECT org_id,uid FROM playlist ORDER BY org_id asc")
```

Comment:
rather than processing results within the SQL results loop, this pulls them into a struct first